src/main/kotlin/com/kotlin/boardproject/domain/comment/repository/LikeCommentRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.kotlin.boardproject.domain.comment.repository
+
+import com.kotlin.boardproject.domain.comment.domain.Comment
+import com.kotlin.boardproject.domain.comment.domain.LikeComment
+import com.kotlin.boardproject.domain.user.domain.User
+import org.springframework.data.jpa.repository.JpaRepository
+
+interface LikeCommentRepository : JpaRepository<LikeComment, Long> {
+    fun findByUserAndComment(user: User, comment: Comment): LikeComment?
```

Comment:
🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline